### PR TITLE
Fix "Variable '${proto}' not found." errors in CI runs

### PR DESCRIPTION
**Pull request by kjplatz**
Fixes the definition of the ${proto} variable in Util.robot.

### CODE REVIEW

**Comment by caglar10ur** [comment]
LGTM :sheep: :it:

File: tests/resources/Util.robot [hunk 1]
```diff
@@ -55,6 +55,16 @@ Get Docker Params
...
     Set Suite Variable  ${vch-ip}  ${ret}
 
+    ${proto}=  Set Variable If  ${certs}  "https"  "http"
+    Set Suite Variable  ${proto}
+
+    :FOR  ${item}  IN  @{output}
+    \   ${status}  ${message}=  Run Keyword And Ignore Error  Should Contain  ${item}  http
+    \   Run Keyword If  '${status}' == 'PASS'  Set Suite Variable  ${line}  ${item}
+
+	${rest}  ${vic-admin}=  Split String From Right  ${line}
```
Comment (mhagen-vmware) [inline tests/resources/Util.robot]: indent again

**Comment by mhagen-vmware** [comment]
minor comment
lgtm

**Comment by mhagen-vmware** [comment]
lgtm

**Comment by kjplatz** [comment]
Bing

**Pull request merged**